client/my-sites/plan-features/header.jsx

```diff
@@ -192,32 +193,22 @@ export class PlanFeaturesHeader extends Component {
 			return <div className={ classes } />;
 		}
 
-		if ( relatedMonthlyPlan ) {
-			const originalPrice = relatedMonthlyPlan.raw_price * 12;
-			return (
-				<span className="plan-features__header-price-group">
-					<PlanPrice
-						currencyCode={ currencyCode }
-						rawPrice={ originalPrice }
-						isInSignup={ isInSignup }
-						original
-					/>
+		if ( discountPrice ) {
+			if ( ! available ) {
+				return (
 					<PlanPrice
-						currencyCode={ currencyCode }
+						currencyCode={ discountPrice }
 						rawPrice={ rawPrice }
 						isInSignup={ isInSignup }
-						discounted
```

Comment:
If we have `discounted` and `original` props on `PlanPrice`, shouldn't we leave the discount calculation to that component (and fix whatever's broken there) rather than leaking that concern into other components?

Reply:
@ockham `PlanPrice` has almost 0 logic, it just displays any `rawPrice` we pass there without . `discounted` and `original` props are not for passing prices but are boolean flags to choose a CSS class name (`original` price is crossed out)